geometry_msgs/Pose object_coordinates
---
string response